src/array/cuda/rowwise_sampling.cu

```diff
@@ -345,10 +345,10 @@ COOMatrix CSRRowWiseSamplingUniform(CSRMatrix mat,
         out_cols,
         out_idxs);
   } else {
-    constexpr int BLOCK_WARPS = 128/WARP_SIZE;
+    constexpr int BLOCK_WARPS = 128/CTA_SIZE;
```

Comment:
I think we need to rename this variable from `BLOCK_WARPS`.

Reply:
yes, maybe ` BLOCK_CTA`?